src/Magnum/GL/Test/AbstractShaderProgramGLTest.cpp

```diff
@@ -257,6 +265,74 @@ void AbstractShaderProgramGLTest::create() {
 
     MAGNUM_VERIFY_NO_GL_ERROR();
     CORRADE_VERIFY(linked);
+    // TODO: decide on this.
+    //CORRADE_VERIFY(program.isLinkFinished());
```

Comment:
To answer your question -- does this return `false` always, once `checkLink()` was called? Or is it random or happening only on some drivers? If you wait half a second ([Utility::System::sleep()](https://doc.magnum.graphics/corrade/namespaceCorrade_1_1Utility_1_1System.html#a7f93179c285d2667b97d9b38476d4cda)), does it eventually become `true`? :D 

If it's random or inconsistent between drivers, then I would do this to have the behavior documented in the test (update the message to say what's actually happening):

```cpp
{
    bool finished = program.isLinkFinished();
    CORRADE_EXPECT_FAIL_IF(!finished, "Some drivers return false GL_COMPLETION_STATUS after the link status has been queried.");
    CORRADE_VERIFY(finished);
} 
```

If it *eventually* becomes `true` after, I'd just add a sleep, if it's not several seconds.